libpod/config/config.go

```diff
@@ -545,5 +545,6 @@ func (c *Config) checkCgroupsAndAdjustConfig() {
 		logrus.Warningf("Alternatively, you can enable lingering with: `loginctl enable-linger %d` (possibly as root)", rootless.GetRootlessUID())
 		logrus.Warningf("Falling back to --cgroup-manager=cgroupfs")
 		c.CgroupManager = define.CgroupfsCgroupsManager
+		c.EventsLogger = "file"
```

Comment:
Are we overriding the config file here? I'm a little concerned that we potentially override explicit configuration from the user.

Reply:
yes, it overrides the config file here.  It works similarly to what we already do with the cgroups manager.

@vrothberg should this check be part of the default file only (`defaultConfigFromMemory`)?

Reply:
I think https://github.com/containers/libpod/blob/master/libpod/config/config.go#L530 (maybe changing the functions name) would be a good place to add that if we wish to _force_ it. If we want users to be able to set it and not overwrite, `defaultConfigFromMemory`.

Reply:
pushed a new version where it is done only for the default configuration